infrastructure/main.tf

```diff
@@ -6,6 +6,8 @@ provider "vault" {
 
 locals {
   app = "bulk-scanning"
+  base_account_name = "${var.product}bulkscan${var.env}"
```

Comment:
Why no hyphens? We separate name components with hyphens by convention.

Reply:
hyphens are not allowed for storage account

Reply:
Fair enough

Reply:
This is one of the limitations for account naming: alphanumerics only, no dashes or underscores

Reply:
I should have read the description